### PR TITLE
Setup Authentication and Dashboard Page

### PR DESCRIPTION
**Pull request by ogwurujohnson**
# Description

This PR, creates the Dashboard component and also tries to implement and consume the signup endpoint by creating reducers and action types

Fixes # (issue)

## Type of change

Please add a relevant label.

## Change status
 - Complete but not tested

# How Has This Been Tested?
 - By running on the browser

Please describe steps taken to ensure this feature is functional and does not break other features:
 - Before making any push I pull from my colleague's branch 

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [x] I have commented my code, particularly in hard-to-understand areas

- [x] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/AddEvent.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+
+export default class AddEvent extends Component {
+    render() {
+        return (
+            <div>
+                <p>Add new event</p>
```
Comment (DianaIgnatescu) [inline src/components/AddEvent.js:7]: Try and have a look at the Seeding to get a sense of what inputs are expected

File: src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default class Login extends React.Component {
+    render(){
+        return(
+            <LoginWrapperStyled> 
+                <h3>Log in here</h3>
+                <div>
+                <input type="text" name="name" placeholder="Name" required />
```
Comment (DianaIgnatescu) [inline src/components/Login.js]: Pay attention to indentation as this can make it harder to read.

File: src/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                </div>
+                <div>
+                <input type="passowrd" name="password" placeholder="Password" required />
```
Comment (DianaIgnatescu) [inline src/components/Login.js]: typo here

File: src/containers/Dashboard.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+export default class Dashboard extends Component {
+    render() {
+        return (
+            <div>
+                <AddEvent />
+                <Events />
+            </div>
+        )
+    }
+}
```
Comment (DianaIgnatescu) [inline src/containers/Dashboard.js]: I think this should be moved inside the Views directory. Same goes for the Login and Register.

**Review by DianaIgnatescu — CHANGES_REQUESTED** [review]
This looks good overall. I have made some small comments that will hopefully help us stay organized since the current structure is a bit difficult to follow.  
Can we please move the actions and reducers directories inside a store directory?
Also the Login and Dashboard should be moved inside the views directory.

File: src/components/AddEvent.js (hunk 1 above)
Comment (ogwurujohnson) [reply src/components/AddEvent.js:7]: fixed

File: src/components/Login.js (hunk 2 above)
Comment (ogwurujohnson) [reply src/components/Login.js]: fixed

File: src/containers/Dashboard.js (hunk 4 above)
Comment (ogwurujohnson) [reply src/containers/Dashboard.js]: fixed

**Review by DianaIgnatescu — APPROVED** [review]
Looks good to me!

**Pull request merged**